custom_components/google_home/config_flow.py

```diff
@@ -106,11 +110,15 @@ def __init__(self, config_entry: ConfigEntry):
         self.config_entry = config_entry
         self.options = dict(config_entry.options)
 
-    async def async_step_init(self, _user_input=None):
+    async def async_step_init(
+        self, _user_input: Optional[Dict[str, Any]] = None
```

Comment:
Do you think we'd be able to replace `Dict[str, Any]]` by custom HA stubs that you have been working on?

Reply:
Not sure, but I can try in another PR.